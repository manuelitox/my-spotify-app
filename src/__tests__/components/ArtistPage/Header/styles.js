import React from 'react'
import { shallow } from 'enzyme'

import StyledHeader from 'components/ArtistPage/Header/styles'

describe('Component | StyledHeader', () => {
  let component 
  beforeEach( () => {
    component = shallow(
      <StyledHeader />
    )
  })
  it('should render a styled component', () => {
    expect(component).toMatchSnapshot()
  })
})