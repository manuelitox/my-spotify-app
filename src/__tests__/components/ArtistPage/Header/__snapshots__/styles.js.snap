// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | StyledHeader should render a styled component 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  background-color: ",
          [Function],
          ";
  display: grid;
  grid-template-columns: 1fr 1fr;
  padding: 10px 20px;
  @media only screen and (max-width: ",
          [Function],
          ") {
    grid-template-columns: 1fr;
  }    
  & > div {
    justify-self: end;
    margin: 0;
    max-width: 420px;
    @media only screen and (max-width: ",
          [Function],
          ") {
      justify-self: start;
      margin-top: 10px;
      padding: 0;
      max-width: 100%;
    }          
    & > div {
      top: 10px;
    }
    input {
      padding: 14px;
    }
  }
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
